python/ray/util/spark/cluster_init.py

```diff
@@ -624,6 +677,8 @@ def _setup_ray_cluster(
             *dashboard_options,
             *_convert_ray_node_options(head_node_options),
         ]
+        if ray_temp_dir is not None:
```

Comment:
when will it be None?

Reply:
When it is in global mode, ray_temp_dir config is None, Ray worker use default temp dir.